data-prepper-plugins/kinesis-source/src/main/java/org/opensearch/dataprepper/plugins/kinesis/source/processor/KinesisRecordProcessor.java

```diff
@@ -177,6 +191,7 @@ public void processRecords(ProcessRecordsInput processRecordsInput) {
             // Flush buffer at the end
             bufferAccumulator.flush();
             recordsProcessed.increment(eventCount);
+            bytesProcessedSummary.record(recordBytes.stream().mapToLong(Integer::longValue).sum());
```

Comment:
These two metrics are the same. We should remove the `bytesReceived` and keep only `bytesProcessed`.